src/locales/ko/move-trigger.ts

```diff
@@ -33,6 +33,7 @@ export const moveTriggers: SimpleTranslationEntries = {
   "isChargingPower": "{{pokemonName}}는(은)\n파워를 모으고 있다!",
   "burnedItselfOut": "{{pokemonName}}의 불꽃은 다 타 버렸다!",
   "startedHeatingUpBeak": "{{pokemonName}}는(은)\n부리를 가열하기 시작했다!",
+  "setUpShellTrap": "{{pokemonName}}는(은)\n트랩셸을 설치했다!",
```

Comment:
```suggestion
  "setUpShellTrap": "{{pokemonName}}[[는]]\n트랩셸을 설치했다!",
```

update `는(은)` -> `[[는]]` following @EnochG1 's suggestion